### ISSUE TITLE
"Invalid URL" error on postman JSON conversion on Windows

### ISSUE THREAD

**Issue by donsequitur** [rangav/thunder-client-support#987]
**Describe the bug**
"Invalid URL" error when "Thunder Client: Convert to Postman" on an exported ThunderClient collection JSON

**To Reproduce**
1. Export a collection to JSON on Windows.  
2. Open exported JSON in VS Code
3. Run "Thunder Client: Convert to Postman" from the command pallete
4. Choose a location to save
5. Observe "Invalid URL" notification from Thunder Client, and no postman json generated

**Expected behavior**
Extension should generate a postman format JSON based on the provided thunderclient JSON.  Failing this it would be nice if the error message could be improved to show what URL is invalid.  I looked in the Thunder Client and generic Extension Host Output panel for any clues but I couldn't find anything

**Platform:** 
 - OS:  Windows
 - vscode version: 1.74.3
 - extension version: 2.3.4

**Your Team Size Using TC:**
1

**Comment by donsequitur** [comment]
Thanks, here is a sample which reproduces the issue for me


```
{
    "client": "Thunder Client",
    "collectionName": "Opensearch",
    "dateExported": "2023-01-25T15:33:44.827Z",
    "version": "1.1",
    "folders": [
        {
            "_id": "3a43562d-a50a-41e8-9ee4-2034a0731ca8",
            "name": "Info",
            "containerId": "",
            "created": "2023-01-20T15:04:47.232Z",
            "sortNum": 5000
        },
        {
            "_id": "a03c7d08-5d52-4d45-9555-405a63d1318e",
            "name": "Snapshot Repository",
            "containerId": "",
            "created": "2023-01-19T20:07:23.201Z",
            "sortNum": 10000
        },
        {
            "_id": "4298bb8e-10a2-4595-91a6-0851f9e3faae",
            "name": "Snapshot",
            "containerId": "",
            "created": "2023-01-19T22:50:48.429Z",
            "sortNum": 20000
        },
        {
            "_id": "35eb1c0e-fca0-4b04-8145-5be89fd0979e",
            "name": "Query",
            "containerId": "",
            "created": "2023-01-23T21:06:36.165Z",
            "sortNum": 30000
        }
    ],
    "requests": [
        {
            "_id": "9aacfdbe-d727-4649-ba6a-02498615fe1b",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "a03c7d08-5d52-4d45-9555-405a63d1318e",
            "name": "Create or update snapshot repository",
            "url": "/_snapshot/{{snapshotRepo}}",
            "method": "PUT",
            "sortNum": 10000,
            "created": "2023-01-19T18:22:54.266Z",
            "modified": "2023-01-20T16:06:50.671Z",
            "headers": [],
            "params": [],
            "body": {
                "type": "json",
                "raw": "{\n  \"type\": \"fs\",\n  \"settings\": {\n    \"location\": \"{{dataDir}}\"\n  }\n}",
                "form": []
            },
            "tests": []
        },
        {
            "_id": "d9a60b75-1490-4cd1-ab0a-c58e15282ac7",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "a03c7d08-5d52-4d45-9555-405a63d1318e",
            "name": "Verify Snapshot Repo",
            "url": "/_snapshot/{{snapshotRepo}}/_verify",
            "method": "POST",
            "sortNum": 20000,
            "created": "2023-01-19T20:02:54.816Z",
            "modified": "2023-01-19T20:07:26.610Z",
            "headers": [],
            "params": [],
            "tests": []
        },
        {
            "_id": "00b0daa4-ff07-4f4a-a33a-9fd3d18306dd",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "a03c7d08-5d52-4d45-9555-405a63d1318e",
            "name": "Get Snapshot Repo Info",
            "url": "/_snapshot/{{snapshotRepo}}",
            "method": "GET",
            "sortNum": 30000,
            "created": "2023-01-19T20:06:48.161Z",
            "modified": "2023-01-19T20:07:27.748Z",
            "headers": [],
            "params": [],
            "tests": []
        },
        {
            "_id": "724860e0-e012-48f5-8136-f281fb6412af",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "4298bb8e-10a2-4595-91a6-0851f9e3faae",
            "name": "Create Snapshot",
            "url": "/_snapshot/{{snapshotRepo}}/{{snapshot}}",
            "method": "PUT",
            "sortNum": 60000,
            "created": "2023-01-19T22:52:15.624Z",
            "modified": "2023-01-19T23:03:24.340Z",
            "headers": [],
            "params": [],
            "body": {
                "type": "json",
                "raw": "{\r\n  \"indices\": \"{{index}}\",\r\n  \"ignore_unavailable\": true,\r\n  \"include_global_state\": false,\r\n  \"metadata\": {\r\n    \"taken_by\": \"donsequitur\",\r\n    \"taken_because\": \"migrating servers\"\r\n  }\r\n}",
                "form": []
            },
            "tests": []
        },
        {
            "_id": "a510e6ed-a7de-47e0-b44f-aa155d028b91",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "4298bb8e-10a2-4595-91a6-0851f9e3faae",
            "name": "Get Snapshot",
            "url": "/_snapshot/{{snapshotRepo}}/{{snapshot}}",
            "method": "GET",
            "sortNum": 70000,
            "created": "2023-01-20T03:30:34.692Z",
            "modified": "2023-01-20T03:31:13.176Z",
            "headers": [],
            "params": [],
            "tests": []
        },
        {
            "_id": "04e96228-0131-4ce4-a962-8eead396b2af",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "4298bb8e-10a2-4595-91a6-0851f9e3faae",
            "name": "Get Snapshot Status",
            "url": "/_snapshot/{{snapshotRepo}}/{{snapshot}}/_status",
            "method": "GET",
            "sortNum": 80000,
            "created": "2023-01-20T03:37:57.623Z",
            "modified": "2023-01-20T03:38:27.969Z",
            "headers": [],
            "params": [],
            "tests": []
        },
        {
            "_id": "57f68799-69bb-408c-a806-a439d445f7aa",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "3a43562d-a50a-41e8-9ee4-2034a0731ca8",
            "name": "Ping",
            "url": "/",
            "method": "GET",
            "sortNum": 90000,
            "created": "2023-01-20T15:05:01.815Z",
            "modified": "2023-01-20T15:05:06.597Z",
            "headers": [],
            "params": [],
            "tests": []
        },
        {
            "_id": "cd910f0d-1456-44c2-bef8-8588196d3cdf",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "4298bb8e-10a2-4595-91a6-0851f9e3faae",
            "name": "Restore Snapshot",
            "url": "/_snapshot/{{snapshotRepo}}/{{snapshot}}/_restore",
            "method": "POST",
            "sortNum": 100000,
            "created": "2023-01-20T17:06:45.784Z",
            "modified": "2023-01-20T17:09:53.176Z",
            "headers": [],
            "params": [],
            "body": {
                "type": "json",
                "raw": "{\n  \"indices\": \"{{index}}\",\n  \"ignore_unavailable\": true,\n  \"include_global_state\": false\n}",
                "form": []
            },
            "tests": []
        },
        {
            "_id": "22b9a4ca-b0c1-44a4-b23b-b2a556c255fc",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "3a43562d-a50a-41e8-9ee4-2034a0731ca8",
            "name": "Cluster Health",
            "url": "/_cluster/health",
            "method": "GET",
            "sortNum": 110000,
            "created": "2023-01-20T17:12:53.321Z",
            "modified": "2023-01-20T17:13:01.206Z",
            "headers": [],
            "params": [],
            "tests": []
        },
        {
            "_id": "bcf28964-8134-42fa-9779-f358e4a91a38",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "3a43562d-a50a-41e8-9ee4-2034a0731ca8",
            "name": "Index Recovery Status",
            "url": "/{{index}}/_recovery",
            "method": "GET",
            "sortNum": 120000,
            "created": "2023-01-20T17:13:30.977Z",
            "modified": "2023-01-20T17:13:43.475Z",
            "headers": [],
            "params": [],
            "tests": []
        },
        {
            "_id": "b810c8b2-ca8d-4f25-8348-99043423d5e8",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "35eb1c0e-fca0-4b04-8145-5be89fd0979e",
            "name": "Search",
            "url": "/crossref/_search",
            "method": "GET",
            "sortNum": 140000,
            "created": "2023-01-23T21:06:42.668Z",
            "modified": "2023-01-23T21:09:32.036Z",
            "headers": [],
            "params": [],
            "body": {
                "type": "json",
                "raw": "{\n  \"size\": 3,\n  \"query\": {\n    \"match\": {\n      \"citation\": {\n        \"query\": \"1. IPCC, Climate Change\"\n      }\n    }\n  },\n  \"rescore\": {\n    \"window_size\": 50,\n    \"query\": {\n      \"rescore_query\": {\n        \"multi_match\": {\n          \"query\": \"1. IPCC Climate Change (2019). 537\",\n          \"type\": \"most_fields\",\n          \"fields\": [\n            \"volume\",\n            \"issue\",\n            \"page\",\n            \"authors\"\n          ]\n        }\n      }\n    }\n  }\n}",
                "form": []
            },
            "tests": []
        },
        {
            "_id": "247b0fa5-35a6-471b-9acd-138a37d8aa20",
            "colId": "1759d99c-5897-4666-8f7b-93e8903dc9a2",
            "containerId": "3a43562d-a50a-41e8-9ee4-2034a0731ca8",
            "name": "Cat Indicies",
            "url": "/_cat/indices",
            "method": "GET",
            "sortNum": 160000,
            "created": "2023-01-25T14:57:39.241Z",
            "modified": "2023-01-25T14:57:57.180Z",
            "headers": [],
            "params": [],
            "tests": []
        }
    ],
    "settings": {
        "headers": [],
        "auth": {
            "type": "basic",
            "basic": {
                "username": "{{username}}",
                "password": "XXXXXXXXXX"
            }
        },
        "tests": [],
        "docs": "redacted",
        "options": {
            "baseUrl": "https://10.10.1.221:9200"
        }
    }
}
```


I imported this to a separate vs code instance and tried converting using both command palette and the dots at the top of the thunder client sidebar, both result in the same error

Appreciate your help.

**Comment by rangav** [comment]
Thanks @donsequitur I can re-produce the error, will fix in the next update.

As a **temporary** fix - please change Ping request from `/` to `/temp`
After conversion, remove the `temp` keyword

<img width="804" alt="Screenshot 2023-01-26 at 09 58 26" src="https://user-images.githubusercontent.com/8637550/214808034-c472374c-9134-4db4-aa2c-f6324b44ab0e.png">

**Comment by rangav** [comment]
This bug is fixed now and published to marketplace, please update to `v2.4.0`

### Thunder Client CLI
- https://github.com/rangav/thunder-client-support/releases/tag/v2.4.0

### Documentation
- For documentation read our blog post on [medium.com](https://rangav.medium.com/d91eb5c71d8e)

### Product Hunt
- We are launching today on [Product Hunt](https://www.producthunt.com/posts/thunder-client-cli), Please **upvote** and let us know any feedback.
